### PR TITLE
add querying and mutating examples

### PR DESCRIPTION
**Pull request by barbalex**
Trying to help another or maybe future me. 

Building on https://github.com/graphile/postgis/issues/30 and https://github.com/graphile/postgis/issues/31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,72 @@ Load the plugin:
...
+      x
+      y
+    }
+  }
+}
+```
+
+In **mutations** `geom_point` is represented as type `GeoJSON`. Example:
+```graphql
+mutation {
```
Comment (benjie) [inline README.md]: I think it'd be clearer (and better practice) if we use a variable for this:

```suggestion
mutation ($id: UUID!, $geomPoint: GeoJSON!) {
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,72 @@ Load the plugin:
...
+  updateDataById(
+    input: {
+      id: "0116254a-0146-11ea-8418-4f89d6596247",
+      dataPatch: {
+        geomPoint: {
```
Comment (benjie) [inline README.md]:
```suggestion
        geomPoint: $geomPoint
```

File: README.md [hunk 3]
```diff
@@ -47,6 +47,72 @@ Load the plugin:
...
+          type: "Point"
+          coordinates: [8.5, 47.5]
+        }
+      }
+    }
+  ) { ... }
+}
+```
+
```
Comment (benjie) [inline README.md]:
````suggestion

with these variables:

```json
{
  "id": "0116254a-0146-11ea-8418-4f89d6596247",
  "geomPoint": {
    "type": "Point",
    "coordinates": [8.5, 47.5]
  }
}
```

````

File: README.md [hunk 4]
```diff
@@ -47,6 +47,72 @@ Load the plugin:
...
+```sql
+CREATE TABLE data (
+  id UUID PRIMARY KEY DEFAULT uuid_generate_v1mc(),
+  geom_point geometry(Point, 4326) default null
+);
+```
+
+In **queries** `geom_point` is represented as type `GeometryPoint`. Example:
+```graphql
+allDatas {
```
Comment (benjie) [inline README.md]: Missing open brace (probably closing brace too?)

```suggestion
{ allDatas {
```

**Comment by barbalex** [comment]
Thanks for your suggestions. What do you think now?

File: README.md [hunk 5]
```diff
@@ -47,6 +47,82 @@ Load the plugin:
...
+In **mutations** `geom_point` is represented as type `GeoJSON`. Example:
+
+```graphql
+mutation ($id: UUID!, $geomPoint: GeoJSON!) {
+  updateDataById(
+    input: {
+      id: $id,
+      dataPatch: {
+        geomPoint: $geomPoint
+        }
```
Comment (benjie) [inline README.md]: We have a bonus closing brace here, I think.
```suggestion
```

Comment (benjie) [reply README.md]: ^ Please commit this, I don't have permission to do so myself.

Comment (barbalex) [reply README.md]: Thanks, is corrected

**Pull request merged**